loomi/regulariza-diagonal-client
# Feature/download data

## DownloadButton

**What I did:**
- Create DownloadButton
- Create mocked pdf
- Add DownloadButton to ResidentsPage, ResidentDetails and ResidentsPending

**How to test:**

1. In your browser go to [stage](http://project.netlify.app/) or [localhost](http://localhost:3000/)
2. Check if the component/view looks just like the [mockup](https://xd.adobe.com/) in different screen sizes (width between 1024px and 1920px)
3. Test the things that you think are worth testing, even the ones that are not in this description

![Captura de tela de 2021-09-09 07-45-45](https://user-images.githubusercontent.com/45184938/132672862-51b7aeff-7771-443e-8a11-6c0f2e8f1e7f.png)
![Captura de tela de 2021-09-09 07-46-06](https://user-images.githubusercontent.com/45184938/132672909-ab088186-f928-4cc8-8cd6-9bbfc3c94ac9.png)
